src/Feliz.MaterialUI/Props.fs

```diff
@@ -1056,15 +1193,6 @@ type buttonBase =
   /// Props applied to the `TouchRipple` element.
   static member inline TouchRippleProps (props: IReactProperty list) = Interop.mkAttr "TouchRippleProps" (createObj !!props)
 
-module buttonBase =
-
-  /// Used to control the button's purpose. This prop passes the value to the `type` attribute of the native button component.
-  [<Erase>]
-  type type' =
-    static member inline submit = Interop.mkAttr "type" "submit"
-    static member inline reset = Interop.mkAttr "type" "reset"
-    static member inline button = Interop.mkAttr "type" "button"
-
```

Comment:
Could you check the MUI repo blame/issues/PRs etc. to make sure this removal isn't a MUI docs bug? This prop is inherited by many components, so it's important we don't remove it unless it's actually intended to be removed.

Reply:
https://material-ui.com/api/button-base/ 

It's still in the docs.

Reply:
I don't see it, try force-refreshing (I often need that to see updated MUI docs; seems they're aggressively cached).